recipes/root/all/conanfile.py

```diff
@@ -0,0 +1,210 @@
+import os
+import stat
+from contextlib import contextmanager
+from glob import glob
+from typing import List
+
+from conans import CMake, ConanFile, tools
+
+
+class PythonOption:
+    OFF = "off"
+    SYSTEM = "system"
+    # in future we may allow the user to specify a version when
+    # libPython is available in Conan Center Index.
+    ALL = [OFF, SYSTEM]
+    DEFAULT = OFF
+
+
+class RootConan(ConanFile):
+    name = "root"
+    version = "v6-22-02"
+    license = "LGPL-2.1-or-later"  # of ROOT itself, the recipe is under MIT license.
+    homepage = "https://root.cern/"
+    # ROOT itself is located at: https://github.com/root-project/root
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "CERN ROOT data analysis framework."
+    topics = ("data-analysis", "physics")
+    settings = ("os", "compiler", "build_type", "arch")
+    options = {
+        # Don't allow static build as it is not supported
+        # see: https://sft.its.cern.ch/jira/browse/ROOT-6446
+        # TODO: shared option should be reinstated when hooks issue is resolved
+        # (see: https://github.com/conan-io/hooks/issues/252)
+        # "shared": [True],
+        "fPIC": [True, False],
+        "python": PythonOption.ALL,
+    }
+    default_options = {
+        # "shared": True,
+        "fPIC": True,
+        "libxml2:shared": True,
+        "sqlite3:shared": True,
+        # default python=off as there is currently no libpython in Conan center
+        "python": PythonOption.OFF,
+    }
+    generators = ("cmake_find_package",)
+    requires = (
+        "opengl/system",
+        "libxml2/2.9.10",
+        "glu/system",
+        "xorg/system",
+        "sqlite3/3.33.0",
+        "libjpeg/9d",
+        "libpng/1.6.37",
+        "libcurl/7.73.0",
+        # "libuuid/1.0.3",
+    )
+
+    @property
+    def _rootsrcdir(self) -> str:
+        version = self.version.replace("v", "")
+        return f"root-{version}"
```

Comment:
f-string are not yet allowed since they are incompatible with python2

Reply:
OK. I will remove these.

Reply:
All f strings and type Python3 style type hints are removed in 7f503ea.